### PR TITLE
UI

### PR DESCRIPTION
**Pull request by Emmykage**
Createed a app.rb file that serves as console app entry-point and has methods that do the following:

    List all books.
    List all people.
    Create a person (teacher or student, not a plain Person).
    Create a book.
    Create a rental.
    List all rentals for a given person id.

defined the entry point, in the main.rb and a  method should do the following:

    Presents the user with a list of options to perform.
    Lets users choose an option.
    Have a way to quit the app.

### CODE REVIEW

File: classes/app.rb [hunk 1]
```diff
@@ -0,0 +1,146 @@
+require_relative 'student'
```
Comment (rustam-tolipov) [inline classes/app.rb:1]: You made great effort by making this project. However, there are some errors that need to be fixed.

File: classes/app.rb [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+require_relative 'teacher'
+require_relative 'rental'
```
Comment (rustam-tolipov) [inline classes/app.rb:3]: If I give a wrong date it is not rising an error. Instead it is breaking the app:

Screen Shot 2022-08-27 at 05 42 47

- [ ] Kindly make sure this error must be fixed to make your code approve!

File: classes/app.rb [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    if @books.empty?
+      puts 'book list is empty'
+    else
+      @books.each { |book| puts "Title: #{book.title} Author: #{book.author}" }
+    end
+  end
+
+  def create_person
+    puts "Create a Teacher or a Student
+        [1] -> Student
```
Comment (rustam-tolipov) [inline classes/app.rb:32]: [OPTIONAL] Rise an error if you give non integer value for Student/Teacher age. As you can here in these images:

TEACHER

> Screen Shot 2022-08-27 at 05 45 02
> 
STUDENT
> Screen Shot 2022-08-27 at 05 44 30
> 
> 
>

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Emmykage, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/app.rb (hunk 3 above)
Comment (rustam-tolipov) [reply classes/app.rb:32]: Hey, I saw your mail! It's an optional requirement you can leave it if you want. But if you want to apply it I would suggest you look for this [link](https://www.geeksforgeeks.org/raising-exceptions-in-ruby/).

**Review by danielufeli — APPROVED** [review]
Hi @Emmykage ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**